### PR TITLE
(plugin) cloud::azure::network::trafficmanager - new plugin

### PR DESCRIPTION
**Pull request by matoy**

### CODE REVIEW

File: cloud/azure/network/trafficmanager/mode/queries.pm [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+use warnings;
+
+sub get_metrics_mapping {
+    my ($self, %options) = @_;
+
+    my $metrics_mapping = {
+        'QpsByEndpoint' => {
+            'output' => 'Queries per second By Endpoint Count',
+            'label'  => 'queries-persecond-count',
+            'nlabel' => 'trafficmanager.queries.persecond.count',
```
Comment (Sims24) [inline cloud/azure/network/trafficmanager/mode/queries.pm:35]: persecond is enough, we usually don't use both.

**Review by Sims24 — CHANGES_REQUESTED** [review]
Thanks once again @matoy few things to change to merge this one.

**Comment by matoy** [comment]
done!

**Comment by Sims24** [comment]
Thank you!

**Comment by Sims24** [comment]
@matoy one last thing, I checked aggregation Microsoft highlights for those metrics and I see that you use Average instead 
image

Any reason for that?

**Comment by matoy** [comment]
no ; just a mistake.
actually average, min ou max would no change the result very much. but total should not be used (works but no interest in this value).

**Review by Sims24 — APPROVED** [review]

**Pull request merged**